qiita_pet/handlers/api_proxy/sample_template.py

```diff
@@ -250,6 +223,41 @@ def sample_template_summary_get_req(samp_id, user_id):
         alert_type = ''
         alert_msg = ''
 
+    return processing, alert_type, alert_msg
+
+
+def sample_template_summary_get_req(samp_id, user_id):
```

Comment:
Github mess - this function already existed, I just pulled out the above function to avoid code duplication. It is already tested.